src/main/java/ru/hogwarts/school/service/AvatarService.java

```diff
@@ -10,18 +12,26 @@
 @Service
 public class AvatarService {
 
+    private static final Logger logger = LoggerFactory.getLogger(AvatarService.class);
```

Comment:
поля static и final надо писать в верхнем регистре
```suggestion
    private static final Logger LOGGER = LoggerFactory.getLogger(AvatarService.class);
```